test/Interop/Cxx/class/safe-interop-mode.swift

```diff
@@ -41,7 +38,7 @@ import CoreFoundation
 func useUnsafeParam(x: Unannotated) { // expected-warning{{reference to unsafe struct 'Unannotated'}}
 }
 
-@available(macOS 13.4, *)
+@available(macOS 13.4, iOS 16.5, watchOS 9.5, tvOS 16.5, *)
```

Comment:
You could also use `@available(SwiftStdlib 5.8, *)`, up to you to decide which one is better 🙂

Reply:
Oh, nice, thanks! I prefer your solution.